cmd/keymasterd/app.go

```diff
@@ -152,7 +152,7 @@ type RuntimeState struct {
 const redirectPath = "/auth/oauth2/callback"
 const secsBetweenCleanup = 30
 const maxAgeU2FVerifySeconds = 30
-const defaultVersionString = "No version provided"
+const defaultVersionString = ""
```

Comment:
I meant to just get rid of this default variable. Later on, compare with the empty string.